jobs/src/main/scala/dcos/metronome/jobrun/impl/JobRunExecutorActor.scala

```diff
@@ -129,8 +128,8 @@ class JobRunExecutorActor(
 
   def tasksFromTaskTracker(): Iterable[JobRunTask] = {
     instanceTracker.specInstancesSync(runSpecId).map(a => a.appTask).collect {
-      case task: LaunchedEphemeral => JobRunTask(task)
-      case task: Task              => throw UnexpectedTaskState(task)
+      case task: Task => JobRunTask(task)
+      case task => throw UnexpectedTaskState(task)
```

Comment:
It seems like a breaking change. Are we sure it should be done this way>

Reply:
LaunchedEphemeral was removed in 1.6 ... I am not sure if this is breaking or not. @meichstedt

Reply:
No that's fine. Before 1.6, we had a `Task` trait and subclasses (`LaunchedEphemeral` and others) to that. 1.6 removed the trait so everything is a Task case class now. The only subclass that would've been a problem was `Reserved` since that's not a launched task but only a reservation representation.